plugin/clone/include/clone_status.h

```diff
@@ -212,6 +212,12 @@ class Status_pfs : public Table_pfs {
     @param[in]	write_error	write error information. */
     void write(bool write_error);
 
+    /** Write to synchronization gtid file.
+    @param[in]	coordinate	synchronization coordinate sent from server.
+    @param[in]  remove      remove the file if exists */
+    void write_synchronization_coordinate(const Key_Value &coordinate,
```

Comment:
The API is overloaded. "remove" is not "write", and the 1st argument gets ignored then. Please split into two. They might need to share a helper to construct the file path.